### DISCUSSION TITLE
Looking for real world redux toolkit example

### DISCUSSION THREAD

**Discussion by kriugen** [reduxjs/redux-toolkit#2381] [Community Recipes]
Hi all,

I am looking for real world application source code that uses redux toolkit. The examples that I managed to find so far are seriously lacking in complexity. I would like to see something with authentication, CRUD, multiple slices with async actions. Especially interesting how to reduce code duplication with addMatcher (or are there other methods?)

Thanks!

**Comment by fregayeg** [comment]
Hello! I'd suggest you to check examples [here](https://github.com/reduxjs/redux-toolkit/tree/master/examples). You can also try on Google **`Redux toolkit open source projects`**. 
If you're using React Native follow this [link](https://github.com/ReactNativeNews/React-Native-Apps)
😉
